wowup-electron/src/app/services/electron/electron.service.ts

```diff
@@ -175,11 +179,33 @@ export class ElectronService {
     return this.invoke(IPC_SET_LOGIN_ITEM_SETTINGS, settings);
   }
 
+  public setAsDefaultProtocolClient() {
+    this.invoke(IPC_SET_AS_DEFAULT_PROTOCOL_CLIENT);
+  }
+
+  public removeAsDefaultProtocolClient() {
+    this.invoke(IPC_REMOVE_AS_DEFAULT_PROTOCOL_CLIENT);
+  }
+
   public async getAppOptions(): Promise<AppOptions> {
     const launchArgs = await this.invoke(IPC_GET_LAUNCH_ARGS);
-    return (<any>minimist(launchArgs.slice(1), {
+    let opts;
+
+    opts = (<any>minimist(launchArgs.slice(1), {
       boolean: ["hidden", "quit"],
+      string: ["install"]
     })) as AppOptions;
+
+    launchArgs.slice(1).forEach(arg => {
```

Comment:
These lines are intended to perform a check for a valid URI in startup arguments. The required parameter is parsed and passed to the import dialog as expected, but this breaks the startup and I cannot see why :/